locale/en/docs/guides/domain-postmortem.md

```diff
@@ -127,16 +127,16 @@ d1.on('error', (er) => { /* handle error */ });
 d1.run(() => setTimeout(() => {
   const d2 = domain.create();
   d2.bar = 43;
-  d2.on('error', (er) => console.error(er.message, domain._stack));
+  d2.on('error', (er) => { console.error(er.message, domain._stack); });
   d2.run(() => {
     setTimeout(() => {
       setTimeout(() => {
         throw new Error('outer');
-      });
-      throw new Error('inner')
-    });
+      }, 1);
```

Comment:
A reader not familiar with how timers work may think that there is a difference between `setTimeout(onTimeout, 0)` and  `setTimeout(onTimeout, 1)` and may wonder why `1` is used here instead of `0`. This is basically the reason for my comment.